portability-spi-transfer/src/main/java/org/datatransferproject/spi/transfer/idempotentexecutor/CachingExecutor.java

```diff
@@ -0,0 +1,92 @@
+package org.datatransferproject.spi.transfer.idempotentexecutor;
```

Comment:
skipping review as I think this belongs to #1079 (feel free to merge in either - but I only reviewed the other copy)